docs/users/cli/account.md

```diff
@@ -0,0 +1,107 @@
+---
+sidebar_position: 2
+---
+
```

Comment:
maybe at the top of this sort of doc we could have a component for prereqs -- 'This tutorial requires `zetacored`. To get this set up, check out this doc'. or something like that

Reply:
https://github.com/zeta-chain/docs/pull/211#:~:text=Add%20a%20note%20about%20installing%20the%20CLI